documentation/BreakingChange-Oad-Rules-Mapping.md

```diff
@@ -0,0 +1,56 @@
+# Azure REST API Breaking Change And Oad Rules Mapping
+
+## Overview
+This specification describes the relationship between [oad rules](https://github.com/Azure/openapi-diff/tree/main/docs) and [breaking change policy](https://aka.ms/AzBreakingChangesPolicy).
+
+## Mapping
+
+| oad rule | change in existing version  | change in new version 
```

Comment:
I think we could simplify this documentation to use a single categorization of:
- breaking change
- change allowed in new api-version